app/helpers/cloud_network_helper/textual_summary.rb

```diff
@@ -8,7 +9,7 @@ module CloudNetworkHelper::TextualSummary
   #
 
   def textual_group_properties
-    TextualGroup.new(_("Properties"), %i[name type status ems_ref])
+    TextualGroup.new(_("Properties"), %i[description type status ems_ref])
```

Comment:
CloudNetwork has both `name` and `description`, any reason to remove the name? (23/30 TextualGroup Properties do show name, 9 show both name & description, I think we can safely show both.)